cmd/show/dashboard.go

```diff
@@ -0,0 +1,323 @@
+				if len(m.newContributorsTable.Rows()) > 0 {
+					contributorName = m.newContributorsTable.SelectedRow()[1]
+				}
+				return m, func() tea.Msg { return SelectMsg{contributorName} }
+			case alumniContributorsView:
+				if len(m.newContributorsTable.Rows()) > 0 {
+					contributorName = m.alumniContributorsTable.SelectedRow()[1]
+				}
+				return m, func() tea.Msg { return SelectMsg{contributorName} }
+			}
+		}
+
+		switch m.tableView {
+		case newContributorsView:
+			m.newContributorsTable, cmd = m.newContributorsTable.Update(msg)
+		case alumniContributorsView:
+			m.alumniContributorsTable, cmd = m.alumniContributorsTable.Update(msg)
+		}
+	}
+
+	return m, cmd
+}
+
+func (m DashboardModel) View() string {
+	return m.drawDashboardView()
+}
+
+// drawTitle: view of PIZZA
+func (m DashboardModel) drawTitle() string {
+	titleRunes1 := []rune{'█', '█', '█', '█', '█', '█', '╗', ' ', '█', '█', '╗', '█', '█', '█', '█', '█', '█', '█', '╗', '█', '█', '█', '█', '█', '█', '█', '╗', ' ', '█', '█', '█', '█', '█', '╗', ' '}
+	titleRunes2 := []rune{'█', '█', '╔', '═', '═', '█', '█', '╗', '█', '█', '║', '╚', '═', '═', '█', '█', '█', '╔', '╝', '╚', '═', '═', '█', '█', '█', '╔', '╝', '█', '█', '╔', '═', '═', '█', '█', '╗'}
+	titleRunes3 := []rune{'█', '█', '█', '█', '█', '█', '╔', '╝', '█', '█', '║', ' ', ' ', '█', '█', '█', '╔', '╝', ' ', ' ', ' ', '█', '█', '█', '╔', '╝', ' ', '█', '█', '█', '█', '█', '█', '█', '║'}
+	titleRunes4 := []rune{'█', '█', '╔', '═', '═', '═', '╝', ' ', '█', '█', '║', ' ', '█', '█', '█', '╔', '╝', ' ', ' ', ' ', '█', '█', '█', '╔', '╝', ' ', ' ', '█', '█', '╔', '═', '═', '█', '█', '║'}
+	titleRunes5 := []rune{'█', '█', '║', ' ', ' ', ' ', ' ', ' ', '█', '█', '║', '█', '█', '█', '█', '█', '█', '█', '╗', '█', '█', '█', '█', '█', '█', '█', '╗', '█', '█', '║', ' ', ' ', '█', '█', '║'}
+	titleRunes6 := []rune{'╚', '═', '╝', ' ', ' ', ' ', ' ', ' ', '╚', '═', '╝', '╚', '═', '═', '═', '═', '═', '═', '╝', '╚', '═', '═', '═', '═', '═', '═', '╝', '╚', '═', '╝', ' ', ' ', '╚', '═', '╝'}
+
+	title1 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes1))
+	title2 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes2))
+	title3 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes3))
+	title4 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes4))
+	title5 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes5))
+	title6 := lipgloss.JoinHorizontal(lipgloss.Left, string(titleRunes6))
+	title := lipgloss.JoinVertical(lipgloss.Center, title1, title2, title3, title4, title5, title6)
+	titleView := lipgloss.NewStyle().Width(m.newContributorsTable.Width()).Margin(2, 60).Foreground(color()).Render(title)
+
+	return titleView
+}
+
+// drawRepositoryTitle: view of the repository info (name, stars, size, and issues)
+func (m DashboardModel) drawRepositoryInfo() string {
+	repoName := lipgloss.NewStyle().Bold(true).AlignHorizontal(lipgloss.Center).Render(fmt.Sprintf("Repository: %s", m.RepositoryInfo.FullName))
```

Comment:
This line paniced when i tried `pizza show freecodecamp/freecodecamp`:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0xc0 pc=0x10071735c]

goroutine 25 [running]:
github.com/open-sauced/pizza-cli/cmd/show.DashboardModel.drawRepositoryInfo({{{{{...}, {...}, 0x0}, {{...}, {...}, 0x0}, {{...}, {...}, 0x0}, {{...}, ...}, ...}, ...}, ...})
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/show/dashboard.go:170 +0x13c
github.com/open-sauced/pizza-cli/cmd/show.DashboardModel.drawDashboardView.func2()
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/show/dashboard.go:220 +0x7c
created by github.com/open-sauced/pizza-cli/cmd/show.DashboardModel.drawDashboardView in goroutine 1
        /Users/jpmcb/workspace/opensauced/pizza-cli/cmd/show/dashboard.go:218 +0x180
```

It also didn't want to return control back to my terminal ... which is weird since the program panicked out.That might be unique to my setup tho since I'm nested deep in tmux usually too so maybe `ctrl-c` was being eaten up somewhere else.

Reply:
Yep, this will indeed panic as it tries to load a repo that is not indexed by the api, and then dereferences nil when loading the dashboard model. This is part of the error flow, handling this in the upcoming changes.

> It also didn't want to return control back to my terminal ... which is weird since the program panicked out.That might be unique to my setup tho since I'm nested deep in tmux usually too so maybe ctrl-c was being eaten up somewhere else.

Yeah, this could be a problem with these setups, which is why bubbletea programs often have 'q', 'esc', and other keymaps to terminate. I know 'esc' gets eaten out when I'm using Vim terminal, so something similar could be happening with tmux.